lib/landrush/win_network_config.rb

```diff
@@ -0,0 +1,159 @@
+require 'ipaddr'
+
+# This class configures the network interface on Windows for use with the Landrush DNS server.
+# It makes use of the netsh executable which is assumed to be available on the Windows host.
+module Landrush
+  class WinNetworkConfig
+    # Windows registry path under which network interface configuration is stored
+    INTERFACES = 'SYSTEM\\CurrentControlSet\\Services\\Tcpip\\Parameters\\Interfaces'.freeze
+
+    def initialize(env={})
+      @env = env
+    end
+
+    # Checks that all required tools are on the PATH and that the Wired AutoConfig service is started
+    def check_prerequisites
```

Comment:
`check_prerequisites` doesn't look like only checking it is starting some service also.

probably `check_and_start_prerequisites`?